index.html

```diff
@@ -73,15 +74,15 @@
 				Inner Right Sidebar
 			</section>
 
-			<ts-note
+			<!-- <ts-note
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)